drivers/nfs.py

```diff
@@ -65,8 +65,7 @@ def check_server_tcp(server, nfsversion=DEFAULT_NFSVERSION):
     False otherwise.
     """
     try:
-        sv = get_supported_nfs_versions(server)
-        return (True if nfsversion in sv else False)
+        return get_supported_nfs_versions(server)
```

Comment:
Yes, which is why I added that comment in NFSSR (around line 116). Nobody seems to be doing anything about the return code. As an alternative, I can call get_supported_nfs_versions in NFSSR  (line 112) directly.